core/templates/pages/splash-page/splash-page.component.ts

```diff
@@ -115,6 +115,10 @@ angular.module('oppia').component('splashPage', {
           // once the controller is migrated to angular.
           $rootScope.$applyAsync();
         });
+
+        $rootScope.$on('$translateChangeSuccess', function() {
```

Comment:
@aks681, are these changes related to any blocking bug? I don't see any info related to these changes in the description. (Note: we aren't cherry-picking the splash page for this release.)

[These changes are resulting in fronted test failure in the release branch.]

Reply:
Oh, this was related to the splash page translate keys being visible, so yeah, it is a required fix, but not for this release, in that case.